oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -65,4 +75,18 @@ protected int getDefaultPort() {
   public String getConnectionType() {
     return connectionType;
   }
+
+  public String getRole() {
+    return role == null ? "normal" : role;
+  }
+
+  @Override
+  public Properties getConnectionArgumentsProperties() {
+    Properties prop = super.getConnectionArgumentsProperties();
+    // To solve the "ORA-01882: timezone region not found" issue, see:
+    // https://stackoverflow.com/questions/9156379/ora-01882-timezone-region-not-found
+    prop.put(TIME_ZONE_AS_REGION_PROPERTY, "false");
+    prop.put(INTERNAL_LOGON_PROPERTY, getRole());
```

Comment:
In the stackoverflow question, I don't see this role needs to be included, why do we need to introduce this variable?

Reply:
This is irrelevant to the stackoverflow question. 

It is a required property, if you are connecting using a SYSDBA or SYSOPR account, you have to specify this property. Actually for those Oracle sql client, you also need to specify the role